bare/main.go

```diff
@@ -5,7 +5,8 @@ import (
 	"os"
 	"os/exec"
 	"path"
-
+	"os/signal"
+	"syscall"
```

Comment:
Could you restore the empty line between the imports of the standard library and those of github.com? The style in the Buildbarn tree is to split imports by 2nd level domain.